dali_tf_plugin/setup.py.in

```diff
@@ -14,16 +14,60 @@
 
 from setuptools import setup, find_namespace_packages
 from dali_tf_plugin_install_tool import InstallerHelper
-from setuptools.command.build_ext import build_ext
 from setuptools.dist import Distribution
 from setuptools.extension import Extension
 import os
+import sys
+
+from setuptools.command.build_ext import build_ext
+from setuptools.command.egg_info import egg_info
+
+
+install_requires = [
+    'nvidia-dali@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@==@DALI_VERSION@'
+]
+
 
 class CustomBuildExt(build_ext, object):
     def run(self):
         helper = InstallerHelper(plugin_dest_dir = os.path.join(self.build_lib, 'nvidia', 'dali_tf_plugin'))
         helper.install()
 
+
+class EggInfoCommand(egg_info):
+    """A class for setuptools egg_info command.
+    Handled by "python setup.py egg_info".
+    """
+
+    @staticmethod
+    def pip_install(*args):
+        pip_args = [
+            "install",
+            "--index-url",
+            "https://developer.download.nvidia.com/compute/redist"
+        ]
+        from pip._internal.cli.main import main as pip
+        return_code = pip(pip_args + install_requires)
+
+        if return_code != 0:
+            raise RuntimeError(
+                "Installation of requirements failed with code: %d" %
+                return_code
+            )
+        else:
+            return return_code
+
+    def run(self):
+        """Run egg_info command."""
+        if all([keyword in sys.argv for keyword in ["egg_info", "--egg-base"]]):
```

Comment:
I think it'd be good to have a comment here saying why we are checking this condition. Is it because we only want to run the first time that egg_info is invoked?